prisma/migrations/20250130165334_create_template_user_view/migration.sql

```diff
@@ -0,0 +1,3 @@
+CREATE VIEW "TemplateUserView" AS
+SELECT "A" as templateId, "B" as userId
+FROM public."_TemplateToUser";
```

Comment:
Just curious if we need the `public` in `public."_TemplateToUser"`?
Quickly looking through the other prisma migrations the generated SQL usually doesn't reference the `public` schema.

Reply:
Looks like you're right!~ Cleaned up.